.github/workflows/schedule.py

```diff
@@ -167,21 +166,32 @@ def create_issue(self, title_date: date) -> None:
         if issues:
             log.info('At least one matching issue already exists: %r', issues)
         else:
-            assignees = self.properties.get('assignees')
-            if assignees is not None:
-                flags.append(f'--assignee={assignees}')
             command = [
                 'gh', 'issue', 'create',
-                *flags,
+                *repo,
                 f'--title={title}',
-                f'--label={self.properties["labels"]}',
                 f'--body={self.body}'
             ]
+            assignees = self.properties.get('assignees', [])
+            if assignees:
+                assignees = [f'--add-assignee={assignees}']
+            labels = self.properties.get('labels', [])
+            if labels:
+                labels = [f'--add-label={labels}']
+            gh_command = ['gh', 'issue', 'edit', *repo]
```

Comment:
```suggestion
            edit_cmd = ['gh', 'issue', 'edit', *repo]
```
`gh_command` is confusing since all the commands are `gh` commands. I recommend using the variables names `list_cmd`, `create_cmd`, and `edit_cmd`.
Also, maybe push this up above setting `assignees` and `labels` so it's right below the create command.